vue-app/src/views/Project.vue

```diff
@@ -96,17 +109,17 @@
           </button>
           <div class="link-box">
             <h2 class="link-title">Check them out</h2>
-            <div class="link-row">
+            <div v-if="project.githubUrl" class="link-row">
               <img src="@/assets/GitHub.svg" />
-              <a href="#">GitHub repo</a>
+              <a :href="project.githubUrl">GitHub repo</a>
             </div>
-            <div class="link-row">
+            <div v-if="project.twitterUrl" class="link-row">
               <img src="@/assets/Twitter.svg" />
-              <a href="#">@Twitter</a>
+              <a :href="project.twitterUrl">@Twitter</a>
             </div>  
-            <div class="link-row">
+            <div v-if="project.websiteUrl" class="link-row">
               <img src="@/assets/Meridians.svg" />
-              <a href="#">www.project.com</a>
+              <a :href="project.websiteUrl">{{ project.websiteUrl }}</a>
             </div>  
           </div>
         </div>
```

Comment:
Below here'll you'll see buttons for "Register", "Add to cart", "In cart", "Received" and "Claim".

At the moment "Add to cart" and "in cart" are pretty clear, but want to work out what the others are doing.